src/Microsoft.Health.Dicom.SqlServer/Features/Workitem/SqlWorkitemStoreV9.cs

```diff
@@ -63,7 +63,9 @@ public virtual async Task<long> BeginAddWorkitemAsync(int partitionKey, DicomDat
 
                 try
                 {
-                    return (long)await sqlCommandWrapper.ExecuteScalarAsync(cancellationToken);
+                    var workitemKey = await sqlCommandWrapper.ExecuteScalarAsync(cancellationToken);
```

Comment:
nit: we can do long workitemKey here and avoid cast in the next line

Reply:
Done